frontend/app/src/components/section-components/users-list-item.tsx

```diff
@@ -1,7 +1,8 @@
 import DropDownButton from './drop-down-button';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faGamepad, faCircle } from '@fortawesome/free-solid-svg-icons';
+import { faGamepad, faCircle, faCrown, faChessKnight } from '@fortawesome/free-solid-svg-icons';
 import {
+  channelRole,
```

Comment:
We could have used context here for passing data deep in the node tree

Reply:
oui most probably